assets/app/scripts/controllers/catalog/images.js

```diff
@@ -25,7 +25,6 @@ angular.module('openshiftConsole')
               imageRepoTag: imageRepoTag,
               name: imageRepo.metadata.name + ":" + imageRepoTag
             };
-            $scope.images.push(image);
```

Comment:
@ncdc does that involve N*M requests, where N is the number of image streams and M is the average number of tags?